roles/core/dhcp_server/molecule/default/molecule.yml

```diff
@@ -0,0 +1,100 @@
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: management1
+    groups:
+      - mg_managements
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+
+    docker_networks:
+      - name: bbnet
+        ipam_config:
+          - subnet: '10.10.0.0/16'
+            gateway: "10.10.0.254"
+    networks:
+      - name: bbnet
+        ipv4_address: "10.10.0.1"            
+
+  - name: c001
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true    
+    networks:
+      - name: bbnet
+        ipv4_address: '10.10.3.1'    
+
+  - name: c002
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true    
+    networks:
+      - name: bbnet
+        ipv4_address: '10.10.3.2'    
+    
+  - name: c003
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true    
+    networks:
+      - name: bbnet
+        ipv4_address: '10.10.3.3'    
+    
+  - name: c004
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true    
+    networks:
+      - name: bbnet
+        ipv4_address: '10.10.3.4'    
+
+  - name: login1
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true    
+    networks:
+      - name: bbnet
+        ipv4_address: '10.10.2.1'    
+
+provisioner:
+  name: ansible
+  inventory:
+    links:
+      hosts: ../../../../../resources/examples/simple_cluster/inventory/cluster/nodes
+      group_vars: ../../../../../resources/examples/simple_cluster/inventory/group_vars
```

Comment:
Please don't use the examples in the tests. Explicitly define each requested variable/additional host instead.

Reply:
Sure I can rework it, but before I just want to understand why we cannot use the examples provided in BB, any specific reason ?
Why we need to rewrite every sigle inventory and needed vars if they already exist in examples ?
Let me know please if this approach is incorrect, because I am thinking to work in some others roles.

Reply:
We are testing the roles, not the examples. Examples must remain examples. If we use these in the tests, they will become de facto standards.

I like to keep each role independent, with all the requested data available in the molecule directory. It's easier to read and debug. All our molecule unit tests are defined this way.

However, this is an interesting use case to test the example inventories. If this is something of interest, I would prefer a dedicated molecule test, global to the project, for each example inventories and playbooks.

Reply:
I have removed the links, now I am declaring a fake host directly in `molecule.yml`
Corrected in commit: 892bd4f